apps/src/codebridge/FileBrowser/StartModeFileDropdownOptions.tsx

```diff
@@ -46,6 +48,10 @@ const StartModeFileDropdownOptions: React.FunctionComponent<
       } else if (type === ProjectFileType.VALIDATION) {
         // If the new type is validation, use the passed all tests validation condition.
         dispatch(setOverrideValidations([PASSED_ALL_TESTS_VALIDATION]));
+        // We also now want to show a banner to levelbuilders to remind them to lock any relevent start files.
+        // We only show the banner for 5 seconds.
+        dispatch(setShowLockedFilesBanner(true));
+        setTimeout(() => dispatch(setShowLockedFilesBanner(false)), 5000);
```

Comment:
nit: from watching the video, 5 seconds feels a little quick for a slow reader like me- could we do 8?

Reply:
can do! 5 was a random choice 😄